conf/modules/distributed_circular_formation.xml

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="distributed_circular_formation" dir="dcf">
+  <doc>
+    <description>Distributed algorithm for circular formations.
+    </description>
+    <section name="Parameters" prefix="DCF_">
+      <define name="MAX_NEIGHBORS" value="4" description="Maximum number of accepted neighbors for an aircraft"/>
+    </section>
+  </doc>
+
+  <settings name="DCF">
+    <dl_settings>
+      <dl_settings NAME="DCF">
+        <dl_settings NAME="Control">
+          <dl_setting MAX="20" MIN="0" STEP="0.2" VAR="dcf_control.k" shortname = "Gain" param="DCF_GAIN_K"/>
+          <dl_setting MAX="200" MIN="0" STEP="1" VAR="dcf_control.radius" shortname = "Radius" param="DCF_RADIUS"/>
+          <dl_setting MAX="5000" MIN="0" STEP="1" VAR="dcf_control.timeout" shortname = "Timeout" param="DCF_TIMEOUT"/>
+        </dl_settings>
+       </dl_settings>
+    </dl_settings>
+  </settings>
+
+  <depends>gvf_module.xml</depends>
+
+  <header>
+    <file name="dcf.h"/>
+  </header>
+
+  <init fun="dcf_init()"/>
+  <!--periodic fun="send_theta_to_nei()" freq="5."/-->
+
+  <datalink message="DCF_REG_TABLE" fun="parseRegTable()"/>
+  <datalink message="DCF_CLEAN_TABLE" fun="parseCleanTable()"/>
+  <datalink message="DCF_THETA" fun="parseThetaTable()"/>
```

Comment:
is it really nessesary to have two messages for reg_table and clean_table. Can't the clean table just be a variable in the reg_table which if set requests that the table be reset? I am just thinking that if we have many messages just containing one variable that the messages list will fill up pretty quick.

Reply:
Additionally, you can combine the REG_TABLE and the THETA message together looking at the sender_id of the message, ground station messages are id 0, other aircraft would be non zero.

Reply:
DCF_THETA should be a single message since it is the only thing that need to be exchange by the aircraft while flying. Better to keep it as simple/minimum as possible.

I agree that I need to think better how to design better the set of messages/payload. The first step was to make the algorithm work. Since I believe that Gautier is thinking about how to deal with or organize the air-to-air messages (in this case they are not really 'datalink'), maybe we can leave  this suggestion for later.